Gruntfile.js

```diff
@@ -109,14 +109,9 @@ module.exports = function(grunt) {
         'Gruntfile.js',
         '__tests__/*.spec.js',
         'src/js/**/*.js',
-        'src/js/**/**/*.js',
-        'src/levels/**/*.js'
```

Comment:
still think hinting the level files is nice, although i guess they are just mostly json

Reply:
Oh, sorry, I think it is `src,js/levels`, but it is `src/levels`, I will fixed it.